system/system_info.go

```diff
@@ -22,7 +22,8 @@ func NewSystemInfo() *SystemInfo {
 	var cmd *exec.Cmd
 	switch runtime.GOOS {
 	case "linux":
-		cmd = exec.Command("uname -a", arg...)
+		arg = append(arg, "-a")
+		cmd = exec.Command("lsb_release", arg...)
```

Comment:
Looks to me like this is handled in this commit: https://github.com/senorprogrammer/wtf/pull/148/files
in PR 148, correct?

Reply:
Yep, you can close this PR if necessary.

Reply:
uname -a is better because it is a core package of linux.